packages/core/types/zapier.generated.d.ts

```diff
@@ -187,6 +187,16 @@ export type FieldChoices =
   | { [k: string]: unknown }
   | (string | FieldChoiceWithLabel)[];
 
+/**
+ * Allows for additional metadata to be stored on the field. Only
+ * simple values are allowed (no objects or arrays)
+ *
+ * [Docs: FieldMetaSchema](https://github.com/zapier/zapier-platform/blob/main/packages/schema/docs/build/schema.md#FieldMetaSchema)
+ */
+export interface FieldMeta {
+  [k: string]: string | number | boolean | null;
```

Comment:
Can we drop `null`? The more strict the better.

```suggestion
  [k: string]: string | number | boolean;
```